### PR TITLE
Mining tut

### PR DESCRIPTION
**Pull request by DanSchrader**
If you check visually, please use mobile view. It's not checked for it's responsive features yet.
I added content to the mining-tutorial page and animated the tutorial articles sliding in and out.

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ export default function NavBar() {
             </LinkStyled>
```
Comment (Ju-Schn) [inline src/components/NavBar.js:43]: Do you need the empty file MiningTutorialSingleLine.js? 🙂

File: src/components/miningtutorial/MinerConfig.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          'Wir entscheiden uns in diesem Tutorial für den Minafacil-Pool, nicht zuletzt, weil er keine Gebühren verlangt und immer gut abschneidet, was das Coin/User-Verhältnis betrifft. Ein Klick auf den minafacil.com-Link in der Tabelle führt uns auch gleich zur Website des Pools.'
+        }
+      />
+      <MiningTutorialImage
+        src={tutorialImage13}
+        alt={'Dokumentationsbild: Miner-Konfiguration'}
+      />
+      <MiningTutorialText
+        text={
+          'Auf der Homepage scrollen wir ein bisschen runter, bis wir zur zum Mining-Konfigurator kommen. Die Stratum-Location setzen wir auf "Europe Stratum". Coin belassen wir bei "Raptoreum" und Solo auf "No".'
```
Comment (Sven-Lyco) [inline src/components/miningtutorial/MinerConfig.js:23]:
```suggestion
          'Auf der Homepage scrollen wir ein bisschen runter, bis wir zum Mining-Konfigurator kommen. Die Stratum-Location setzen wir auf "Europe Stratum". Coin belassen wir bei "Raptoreum" und Solo auf "No".'
```

File: src/components/miningtutorial/Hint.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        }
+      />
+      <MiningTutorialText
+        text={
+          'Die Strompreise sind hoch und die Werte der meisten Kryptowährungen aufgrund von Marktkorrekturen und politischen Auslösern drastisch gefallen.'
+        }
+      />
+      <MiningTutorialText
+        text={
+          'Minen sollte zur Zeit nur, wer an die Zukunft des Coins glaubt, den er minen möchte. Aufgrund der Ethereum-Umstellung auf PoS, kann CPU-Mining jedoch durchaus profitabel werden. Es ist eine Frage der eigenen Einschätzung und jedermanns eigenes Risiko.'
```
Comment (Ju-Schn) [inline src/components/miningtutorial/Hint.js:22]: Really nice German 👍

File: src/components/miningtutorial/MiningTutorial.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+import MinerConfig from './MinerConfig';
+import MinerSetup from './MinerSetup';
+
+export default function MiningTutorial({
+  showTutorialArticle,
+  hideTutorialArticle,
+  tutorial,
+}) {
+  return (
+    <TutorialArticleList isVisible={showTutorialArticle}>
```
Comment (Sven-Lyco) [inline src/components/miningtutorial/MiningTutorial.js:18]:
```suggestion
    <TutorialArticleList isVisible={showTutorialArticle} role="list">
```

File: src/components/miningtutorial/TecSpecs.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  return (
+    <TutorialArticleItem>
+      <MiningTutorialHeadline headline={'Technische Anforderungen'} />
+      <MiningTutorialText
+        text={
+          'Für dieses Tutorial gehen wir von einer weit verbreiteten Hard- und Softwarekombination aus, die für die meisten Otto-Normalverbraucher relevant und für das Mining selbst auch mit am Profitabelsten ist:'
+        }
+      />
+      <TechSpecList>
+        <TechSpecListItem>Desktop-PC</TechSpecListItem>
```
Comment (Ju-Schn) [inline src/components/miningtutorial/TecSpecs.js:16]: Do you need a key prop here? you could use Math.random() * Math.random() ore nanoid()

File: src/pages/MiningTutPage.js [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+  const handleHideTutorialArticle = () => {
+    setShowTutorialArticle(false);
+  };
+
+  return (
+    <MiningTutWrapper>
+      <ScrollToTop />
+      <MiningTutTitle>Mining-Tutorial</MiningTutTitle>
+      <HeadlineList>
```
Comment (Sven-Lyco) [inline src/pages/MiningTutPage.js:23]:
```suggestion
      <HeadlineList role="list">
```

**Review by Ju-Schn — APPROVED** [review]

**Review by Sven-Lyco — COMMENTED** [review]
Looks good to me, just a few suggestions :)

File: src/components/miningtutorial/MiningTutorial.js (hunk 4 above)
Comment (DanSchrader) [reply src/components/miningtutorial/MiningTutorial.js:18]: Great thanks to you, my dear!

**Pull request merged**